### ISSUE TITLE
On iOS, when the text in a new post pushes attached imaged below the viewable area, I cannot add alt text.

### ISSUE THREAD

**Issue by adamatic-me** [bluesky-social/social-app#1816]
**Describe the bug**
On iOS, when the text in a new post pushes attached imaged below the viewable area, I cannot add alt text.

**To Reproduce**

1. On iOS, create a Post.
2. Add an image.
3. Try to post, be prevented from posting, because your settings prevent no alt text image posts
4. Notice that you cannot scroll down enough to see the image, and add the alt text.
5. Delete some text from the post, then you can scroll to the image and add the alt text.

**Expected behavior**

Allow scrolling down enough to see the attached image, at add alt text.

**Screenshots**

https://github.com/bluesky-social/social-app/assets/146130101/b69d66e4-deb3-40cf-b325-2ffe56acf3a1

**Details**

 - Platform: iOS
 - Platform version: 17.0.3
 - App version: 1.55.0
